@here/harp-examples/src/markers_screen-anchor.ts

```diff
@@ -162,19 +181,14 @@ export namespace GeoToScreenExample {
 
     function addInfoMessage() {
         const message = document.createElement("div");
+        message.className = "message";
         message.innerHTML = `
   <br />  This example shows how MapView.getScreenPosition works for various cases
   <br />     the red square is painted on the screen in screen coordinates, whereas the green cube
   <br />     lives in the world.
   <br />  Use the arrow keys or the gui to change the geoPosition
   <br />  Jump to next worlds with "j" and "l"
   `;
```

Comment:
I'd remove those `<br />` elements, add proper punctuation and let the browser to flow the text.

Reply:
Good point, added paragraphs to separate the two different sections.